src/client/components/app/Transactions/Labs/LabDepositTx.tsx

```diff
@@ -149,9 +157,8 @@ export const LabDepositTx: FC<LabDepositTxProps> = ({ onClose }) => {
       expectedTxOutcomeStatus.error ||
       actionsStatus.approveDeposit.error ||
       actionsStatus.deposit.error ||
-      slippageError ||
-      tokenAllowanceError,
-    loading: expectedTxOutcomeStatus.loading || isDebouncePending || isLoadingTokenAllowance,
+      slippageError,
+    loading: expectedTxOutcomeStatus.loading || isDebouncePending,
```

Comment:
When `isFetchingAllowance` is not loading for the target status?

Reply:
it just disables the approve button when it is loading now, imo it should not affect the simulation output from showing, since either it is approved or not, but target expected outcome persists anyway